app/src/main/java/com/bencarlisle15/terminalhomelauncher/managers/xml/options/Ui.java

```diff
@@ -971,6 +971,22 @@ public String type() {
         public String info() {
             return "[horizontal_margin],[vertical_margin],[horizontal_padding],[vertical_padding]";
         }
+    },
+    status_borders {
+        @Override
+        public String defaultValue() {
+            return "false";
+        }
+
+        @Override
+        public String info() {
+            return "If true, the status area will contain a border of color status_bordercolor.";
```

Comment:
This should probably be changed to `status_bgrectcolor` instead of `status_bordercolor`